docs/contributing.rst

```diff
@@ -250,11 +250,8 @@ More docs on this coming soon...
 
 .. _Good first issue: https://github.com/modin-project/modin/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue+%3Abeginner%3A%22
 .. _Documentation: https://github.com/modin-project/modin/issues?q=is%3Aissue+is%3Aopen+label%3A%22documentation+%3Abookmark_tabs%3A%22
-.. _architecture: architecture.html
-.. _internal methods:
 .. _black: https://github.com/ambv/black
 .. _flake8: http://flake8.pycqa.org/en/latest/
 .. _Github Actions: https://github.com/features/actions
-.. _testing:
```

Comment:
Why is this removed?

Reply:
Because this link opens non-valid page for `testing` (this [page](https://github.com/airspeed-velocity/asv#airspeed-velocity), which are the next in links-list ).
![image](https://user-images.githubusercontent.com/26389717/106130705-bec9e300-6172-11eb-8101-c6a9f3d5bae4.png)

Reply:
Probably, we need to add a documentation page for testing because we don't have it. Can you open an issue for this?

Reply:
Issue #2661  was created

Reply:
Thanks!